sweeper.go

```diff
@@ -29,10 +39,17 @@ type sweeper struct {
 
 	lastUpdateTime  time.Time
 	registerChannel chan *Meter
+
+	clock clock.Clock
 }
 
 func (sw *sweeper) start() {
 	sw.registerChannel = make(chan *Meter, 16)
+	if globalClock == nil {
```

Comment:
any reason not to initialize the global clock to `clock.New()`?

Reply:
Premature optimization, I guess. We can totally pay the price of `clock.New` in case we set a mock clock.